apps/leo_manager/src/leo_manager_formatter_text.erl

```diff
@@ -1099,14 +1099,13 @@ bucket_by_access_key(Buckets) ->
                   BucketStr = binary_to_list(Bucket1),
                   PermissionsStr = leo_s3_bucket:aclinfo_to_str(Permissions1),
 
-                  RedOptions = leo_redundant_manager_api:get_options(),
+                  {ok, RedOptions} = leo_redundant_manager_api:get_options(),
```

Comment:
I tried `leofs-adm get-bucket 05236` with one bucket named `test` and got the result as below.
```shell
$ ./leofs-adm get-bucket 05236
bucket   | permissions      | redundancy method        | created at
---------+------------------+--------------------------+---------------------------
test     | Me(full_control) | copy, {n:1, w:1, r:1, d: | 2017-04-11 09:24:43 +0900
```
The column `redundancy method` for `test` is not fully displayed so need to pad additional spaces for it.